doc/manual/manual.bib

```diff
@@ -4186,16 +4212,25 @@ @Article{HR00a
   OPTannote =    {}
 }
 
-@InCollection{HR00b,
-  author =       {Heuveline, V. and Rannacher, R.},
-  title =        {A numerical tool for flow simulation in a {W}ankel motor},
-  booktitle =    {Mathematik: Schl{\"u}sseltechnologie f{\"u}r die Zukunft},
-  publisher =    {Springer, Berlin-Heidelberg},
-  year =         2000,
-  editor =       {Hoffmann, K. -H. et al.},
-  note =         {to appear}
+
+@Inbook{HR00b,
+  author="Rannacher, Rolf
+  and Heuveline, Vincent",
+  editor="J{\"a}ger, Willi
+  and Krebs, Hans-Joachim",
+  title = "A Numerical Tool for Flow Simulation in a Wankel Motor",
```

Comment:
again, I would prefer {} over ""

Reply:
These come free with some citation exporters.

Reply:
That's an example of a publication not worth updating, given that it clearly has no relevance to us...